src/components/Gallery.js

```diff
@@ -30,45 +39,55 @@ const AboutContainer = ({ author, description, license, filePage, close }) => {
   return (
     <div class='gallery-about' ref={containerRef}>
       <div class='header'>{i18n('about-header')}</div>
-      {
-        description && (
-          <div>
-            <div class='sub-header'>{i18n('gallery-description')}</div>
-            <p class='description'>{description}</p>
-          </div>
-        )
-      }
-      {
-        (author || license) && (
-          <div>
-            <div class='sub-header'>{i18n('gallery-author-license')}</div>
-            <p>
-              {author}{ author && <br /> }
-              {license}
-            </p>
-          </div>
-        )
-      }
+      <div>
+        <div class='sub-header'>{i18n('gallery-description')}</div>
+        <p class='description'>{mediaInfo.description || caption || title}</p>
```

Comment:
We are always guaranteed to have at least `title` here, correct? That's why there's no need for a `gallery-unknown-description` label?

Reply:
Yes